cmd/manager/main.go

```diff
@@ -266,12 +266,6 @@ func serveCRMetrics(cfg *rest.Config, operatorNs string) error {
 }
 
 func printConfig(cfg *configuration.Config) {
-	logWithValuesRegServ := log
```

Comment:
I think these could be useful to print. Let's maybe just mention that those variables can be overridden on the reg-service site and different values may be used by the reg-service instead.

Reply:
removed here: 0bd59fa6669e3cf716c0411dd11adc2b9261ba8b

Reply:
done here: a2b8e79b0b05c4daed057b301cf845fe7469c6c6

Reply:
@alexeykazakov i've also moved this to be part of the configuration (same as what you had suggested in the registration-service)